### PR TITLE
feat: Embed Join Us & Sponsor Us Forms | Update Date Component

### PR DESCRIPTION
**Pull request by ratnakar5938**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [ ] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: src/components/HeroSection/JoinUs.jsx [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            <JoinHeading>{join.JoinUs.heading}</JoinHeading>
+            <JoinDescription>{join.JoinUs.description}</JoinDescription>
+            <ButtonContainer>Join Us</ButtonContainer>
+          </JoinUsText>
+        </JoinUsDiv>
+        <SponsorDiv>
+          <SponsorHeading>{join.SponsorUs.heading}</SponsorHeading>
+          <SponsorDescription>{join.SponsorUs.description}</SponsorDescription>
+          <SponsorImg src={join.SponsorUs.img.src} alt={join.SponsorUs.img.alt} />
+          <ButtonContainer>Sponsor Us</ButtonContainer>
```
Comment (riteshsp2000) [inline src/components/HeroSection/JoinUs.jsx:140]: Sponsor us button is not working

File: src/components/HeroSection/JoinUs.jsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  return (
+    <Container>
+      <Wrapper>
+        <JoinUsDiv>
+          <JoinUsImg src={join.JoinUs.img.src} alt={join.JoinUs.img.alt} />
+          <JoinUsText>
+            <Head>{join.JoinUs.head}</Head>
+            <JoinHeading>{join.JoinUs.heading}</JoinHeading>
+            <JoinDescription>{join.JoinUs.description}</JoinDescription>
+            <ButtonContainer>Join Us</ButtonContainer>
```
Comment (riteshsp2000) [inline src/components/HeroSection/JoinUs.jsx:133]: Join us button is not working.

**Review by riteshsp2000 — CHANGES_REQUESTED** [review]

**Comment by riteshsp2000** [comment]
Screenshot 2021-09-11 at 4 32 05 PM


Hey, @ratnakar5938 The image is having issues with the top corner border-radius on mobile view. Please fix them

**Comment by riteshsp2000** [comment]
Screenshot 2021-09-11 at 4 33 05 PM

For desktop view, please make sure that the text `Be a hack mentor` is aligned left with the rest of the texts. For mobile it should be centered.

File: src/components/HeroSection/JoinUs.jsx (hunk 1 above)
Comment (riteshsp2000) [reply src/components/HeroSection/JoinUs.jsx]: You can find the links for both the typeforms in config/website.js

Comment (riteshsp2000) [reply src/components/HeroSection/JoinUs.jsx]: @abhibhaw @nayakastha what do you think? should we embed Typeform in the website which pops up on clicking the buttons? or should we redirect them on other tab?

Comment (riteshsp2000) [reply src/components/HeroSection/JoinUs.jsx]: I personally feel that we should embed them like a pop up

**Comment by riteshsp2000** [comment]
@ratnakar5938 Also make sure that you update the content.
1. Join us: Wanna help people, out of the experiences and the knowledge you have? Then, this is your chance, be a mentor and help people in their hackathon journey!
2. Sponsor us: Join us and be a part of the hackathon, which is one of its own kind! Let's make it bigger and better, together.

This will go in place of the lorem ipsum text.

File: config/content/Hero.js [hunk 3]
```diff
@@ -1,6 +1,9 @@
 /* eslint-disable max-len */
 export default {
-  dates: '29th to 31st October 2021',
+  dates: {
+    src: 'https://res.cloudinary.com/dscnitrourkela/image/upload/project-oregano/assets/xoptbvvefk1jrkb5pjnp.png',
+    alt: 'hackNITR dates',
```
Comment (DesignrKnight) [inline config/content/Hero.js:5]:
```suggestion
    alt: 'Image representing dates of the HackNITR Hackathon happening from 28-31 October 2021',
```

Comment (DesignrKnight) [reply config/content/Hero.js:5]: @riteshsp2000 @CIPHERTron do confirm dates before commit

File: config/content/Join.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable max-len */
+export default {
+  JoinUs: {
+    head:'Be a hack mentor',
+    img: {
+      src: 'https://res.cloudinary.com/dscnitrourkela/image/upload/project-oregano/assets/qt8zgt5yhmsdhqo5qux7.gif',
+      alt: 'Join Us',
+    },
+    heading: 'Be the mentor you were meant to be!',
+    description: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Delectus at provident',
```
Comment (DesignrKnight) [inline config/content/Join.js:10]: Either add relevant content here or remove it. And similarly below

File: config/content/Hero.js (hunk 3 above)
Comment (CIPHERTron) [reply config/content/Hero.js:5]: Yeah sure! 💪🏻

File: config/content/Join.js (hunk 4 above)
Comment (CIPHERTron) [reply config/content/Join.js:10]: Noted.

File: src/components/HeroSection/JoinUs.jsx (hunk 1 above)
Comment (abhibhaw) [reply src/components/HeroSection/JoinUs.jsx]: embedding is better

Comment (abhibhaw) [reply src/components/HeroSection/JoinUs.jsx]: That will increase the page view time, better SEO :)

**Comment by riteshsp2000** [comment]
@ratnakar5938 
Use react-typeform-embed package to embed both the forms like a popup. Check out the code in the repo given below for reference.
https://github.com/TEDxNITRourkela/website/blob/3a2a3790e48e2c7c45fe74c9413f2521a5f2e8e4/client/src/components/homepage/index.js#L52

- The desired behaviour is that, on clicking the join us button, the corresponding typeform should popup and similarly do the same for sponsor us.
- No need to create a separate modal/popup component, the above mentioned package provides everything that is needed.

**Review by riteshsp2000 — APPROVED** [review]

**Review by nayakastha — APPROVED** [review]

**Pull request merged**